app/helpers/admin/content_data_routes_helper.rb

```diff
@@ -1,5 +1,16 @@
 module Admin::ContentDataRoutesHelper
+  def content_data_home_url
+    @content_data_home_url ||= "#{content_data_base_url}/content"
+  end
+
+  def content_data_page_data_url(edition)
+    path = public_document_path(edition)
+    @content_data_page_data_url ||= "#{content_data_base_url}/metrics#{path}"
```

Comment:
I'm not sure what value the memoization adds here. `content_data_base_url` is memoized, and `path` is being computed every time, so the only computation being saved is building the combined string.

Currently, it looks to me like this function will only work for one edition, and after that, it'll return the same value regardless of the argument you provide.